microsoft-support.md

```diff
@@ -1,61 +1,14 @@
-# Microsoft .NET Core support lifecycle
+# Microsoft support for .NET
 
-Every Microsoft product has a lifecycle, including .NET Core. The lifecycle begins when a product is released and ends when it's no longer supported. Knowing key dates in this lifecycle helps you make informed decisions about when to upgrade or make other changes to your software. This product is governed by the [Microsoft Modern Lifecycle](https://support.microsoft.com/help/30881/modern-lifecycle-policy).
+Microsoft supports .NET on multiple operating systems, per the [Microsoft Modern Lifecycle](https://support.microsoft.com/help/30881/modern-lifecycle-policy), [.NET release lifecycle](releases.md), and [operating system lifecycle](os-lifecycle-policy.md).
 
-This document describes the support lifecycle for: .NET Core, ASP.NET Core and EF Core.
+Support is provided for: .NET, ASP.NET Core and EF Core.
 
-The [.NET Core OS Lifecycle Policy](https://github.com/dotnet/core/blob/master/os-lifecycle-policy.md) describes support for various operating systems.
+The .NET lifecycle for a new version begins when it is released and ends when it's no longer supported. Knowing key dates for a product lifecycle helps you make informed decisions about when to upgrade or make other changes to your software and computing environment.
```

Comment:
First sentence is an awkward tautology. I think you're trying to describe that each 'new version' (also a little fraught given our array of versions) has a distinct lifecycle and reference the importance of understand the dates. I expected to see something related to the dates following this section but realized that that info is hiding behind links presented in the opening sentence. Perhaps also link https://github.com/dotnet/core/blob/8d097ce73299c4632beee35ec9ff67c401d89bdb/releases.md in the 'knowing key dates' sentence?

Reply:
Yes, it is awkward. Good call. This was the original text: "Every Microsoft product has a lifecycle, including .NET Core. The lifecycle begins when a product is released and ends when it's no longer supported."

Took another run at it with: https://github.com/dotnet/core/pull/5819/commits/a1518ee9dae20d858a9660375531a5bae548aebb. Better?